pkg/git/git.go

```diff
@@ -335,7 +332,7 @@ func (r *Repo) MergeBase(from, to string) (string, error) {
 	commitRevs := []string{masterRef, releaseRef}
 	var res []*object.Commit
 
-	var hashes []*plumbing.Hash
+	hashes := []*plumbing.Hash{}
```

Comment:
`prealloc`?

Reply:
Yes, I also thought that I read somewhere that this is the more idiomatic way for golang.

Reply:
Cool! Just my curiosity; not calling out an issue or anything. :)